receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/accumulator.go

```diff
@@ -109,3 +114,19 @@ func overrideWithTaskLevelLimit(taskMetrics *ECSMetrics, metadata TaskMetadata)
 		taskMetrics.CPUUtilized = ((taskMetrics.CPUUsageInVCPU / taskMetrics.CPUReserved) * 100)
 	}
 }
+
+func calculateTime(startTime, endTime string) (float64, error) {
+	const layout = "2006-01-02T15:04:05"
+	startTime = strings.Trim(startTime, "Z")
+	endTime = strings.Trim(endTime, "Z")
+	start, err1 := time.Parse(layout, startTime)
```

Comment:
nit: I think we can use same variable, `err`.

Reply:
Sure